app/assets/stylesheets/shared.scss

```diff
@@ -184,12 +184,17 @@
 
 .global-notice {
   font-family: var(--ff-sans-serif);
-  background: var(--accent-success-darker);
-  color: white;
+  background: var(--global-notice-bg);
+  color: var(--global-notice-text);
   padding: 20px 0px 20px;
   text-align: center;
   position: relative;
   top: 0px;
   left: 0px;
   right: 0px;
 }
+
+.global-notice a {
+  color: var(--global-notice-link);
+  text-decoration: underline;
+}
```

Comment:
With changes suggested in previous comments on _colors.scss, you should be able to drop this part of code entirely :)